asset_path_tag.rb

```diff
@@ -1,16 +1,25 @@
 # Title: Asset path tag for Jekyll
-# Author: Sam Rayner http://samrayner.com
+# Authors:
+#     Sam Rayner http://samrayner.com
+#     Otto Urpelainen http://koti.kapsi.fi/oturpe/projects/
+#
 # Description: Output a relative URL for assets based on the post or page
 #
-# Syntax {% asset_path [filename] %}
+# Syntax
+#    {% asset_path filename [post_id] %}
+#    {% asset_path "filename with whitespace" [post_id] %}
 #
 # Examples:
 # {% asset_path kitten.png %} on post 2013-01-01-post-title
 # {% asset_path pirate.mov %} on page page-title
+# {% asset_path document.pdf /2012/05/25/another-post-title %}
+# {% asset_path "document with long name.pdf" /2012/05/25/another-post-title %}
```

Comment:
`# {% asset_path "document with spaces in name.pdf" /2012/05/25/another-post-title %}`